spk/dante-sockd/src/sockd.conf

```diff
@@ -0,0 +1,39 @@
+# sockd configuration
+# https://www.inet.no/dante/doc/1.4.x/sockd.conf.5.html
+
+logoutput: @socks_log@
+
+internal: eth0 port = @socks_port@
```

Comment:
I wonder if configurable "socks_port" should be opened in firewall.
If so, it may be interested that `service_preinstall` generates `conf/dante-sockd.sc` so that it can be registered as a known service port in DSM: https://github.com/SynoCommunity/spksrc/blob/master/mk/spksrc.service.installer#L245
What do you think about it?

Reply:
I think, if you are using firewall, you can open port manually, if needed. It's not a problem.